arch/arm64/src/common/arm64_backtrace.c

```diff
@@ -160,8 +160,8 @@ int up_backtrace(struct tcb_s *tcb,
     {
       ret = backtrace(tcb->stack_base_ptr,
                       tcb->stack_base_ptr + tcb->adj_stack_size,
-                      (void *)up_current_regs()[REG_X29],
-                      (void *)up_current_regs()[REG_ELR],
+                      (void *)running_regs()[REG_X29],
+                      (void *)running_regs()[REG_ELR],
```

Comment:
could remove `(void *)` with the new prototype

Reply:
ok